src/app/Classes/LogViewer.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Backpack\LogManager\app\Classes;
+use Illuminate\Support\Facades\Storage;
+
+/***
+ * Class LogViewer
+ * @package Backpack\LogManager\app\Classes
+ *
+ * Credits https://github.com/rap2hpoutre/laravel-log-viewer
+ */
+
+class LogViewer
+{
+    /**
+     * @var string file
+     */
+    private static $file;
+
+    private static $levels_classes = [
+        'debug'     => 'info',
```

Comment:
Can we explain (in a comment) why `debug` is `info` etc? It sounds like _NewSpeak_ to me.